amd_openvx_extensions/amd_rpp/source/Downmix.cpp

```diff
@@ -0,0 +1,271 @@
+/*
+Copyright (c) 2019 - 2022 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+#include "vx_ext_amd.h"
+#define NUM_OF_DIMS 5
+
+struct DownmixLocalData
+{
+    RPPCommonHandle *handle;
+    Rpp32u device_type;
+    Rpp32u nbatchSize;
+    RppPtr_t pSrc;
+    RppPtr_t pDst;
+    vx_int32 *samples;
+    vx_int32 *channels;
+    RpptDescPtr src_desc_ptr;
+    RpptDesc srcDesc;
+    RpptDesc dstDesc;
+    RpptDescPtr dst_desc_ptr;
+    size_t in_tensor_dims[NUM_OF_DIMS];
```

Comment:
Please change the naming style of parameter from snake case to camel case
Applicable to all such instances in the file
Please refer Brightness.cpp for the naming convention followed for the variables
https://github.com/swetha097/MIVisionX/blob/swbs/audio/pr1/decoder/amd_openvx_extensions/amd_rpp/source/Brightness.cpp

Reply:
Changed the naming conventions